packages/core/strapi/src/commands/actions/plugin/link-watch/action.ts

```diff
@@ -0,0 +1,98 @@
+import boxen from 'boxen';
+import chalk from 'chalk';
+import concurrently from 'concurrently';
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import nodemon from 'nodemon';
+import { outdent } from 'outdent';
+
+import { notifyExperimentalCommand } from '../../../utils/helpers';
+import { CLIContext } from '../../../types';
+import { loadPkg, validatePkg } from '../../../utils/pkg';
+
+interface ActionOptions {}
+
+export default async (_opts: ActionOptions, _cmd: unknown, { cwd, logger }: CLIContext) => {
+  try {
+    /**
+     * Notify users this is an experimental command.
+     */
+    await notifyExperimentalCommand('plugin:watch:link', { force: true });
+
+    const outDir = './dist';
+    const extensions = 'ts,js,png,svg,gif,jpeg,css';
+
+    nodemon({
+      watch: [outDir],
+      ext: extensions,
+      exec: 'yalc push --changed',
+    });
+
+    const folder = path.join(cwd, outDir);
+
+    if (!(await pathExists(folder))) {
+      await fs.mkdir(folder);
+    }
+
+    const pkg = await loadPkg({ cwd, logger });
+    const pkgJson = await validatePkg({ pkg });
+
+    concurrently(['npm run watch']);
+
+    nodemon
+      .on('start', () => {
+        logger.info(
+          outdent`
+        Watching ${outDir} for changes to files with extensions: ${extensions}
+
+        To use this package in Strapi, in a separate shell run:
+        cd /path/to/strapi/project
+
+        Then run one of the commands below based on the package manager used in that project:
+
+        ## yarn
+        ${chalk.greenBright(`yalc add --link ${pkgJson.name} && yarn install`)}
```

Comment:
I found I needed to do `yarn dlx yalc add --link NAME && yarn install` Should we change this?

Reply:
yes 😄